go/vt/vttablet/tabletmanager/vreplication/controller.go

```diff
@@ -305,7 +305,7 @@ func (ct *controller) runBlp(ctx context.Context) (err error) {
 		if (err != nil && vr.WorkflowSubType == int32(binlogdatapb.VReplicationWorkflowSubType_AtomicCopy)) ||
 			isUnrecoverableError(err) ||
 			!ct.lastWorkflowError.ShouldRetry() {
-
+			err = vterrors.Wrapf(err, "terminal error")
```

Comment:
Worth making `terminal error` an exported const IMO. Then that can be used in the query as well: `OR locate ('terminal error:', message) = 1` and in the unit test.

Reply:
I've parameterized the string, using the parameter in creation and in SQL. However, in unit tests I'm still using the hard coded text because I think it makes the unit test clearer.